### PR TITLE
Terminology Updated

### PR DESCRIPTION
**Pull request by jivteshsingh**
# Problem

   Consistent ListenBrainz Terminology - Implemented New Terminology across all pages of the Web App.

# Solution

Tried to change recording with track and release with album.

# Action

  Implemented all the changes and now everywhere track and abum is rendered instead of recording and release/release groups :)

### CODE REVIEW

File: listenbrainz/webserver/static/js/src/stats/UserTopEntity.tsx [hunk 1]
```diff
@@ -14,6 +14,7 @@ export type UserTopEntityProps = {
   entity: Entity;
   user?: ListenBrainzUser;
   apiUrl: string;
+  newTerminology: string;
```
Comment (amCap1712) [inline listenbrainz/webserver/static/js/src/stats/UserTopEntity.tsx:17]: just `terminology` as the variable name should be fine.

File: listenbrainz/webserver/static/js/src/stats/UserEntityChart.tsx [hunk 2]
```diff
@@ -46,6 +46,7 @@ export type UserEntityChartState = {
   listenContainerHeight?: number;
   hasError: boolean;
   errorMessage: string;
+  Terminology: string;
```
Comment (amCap1712) [inline listenbrainz/webserver/static/js/src/stats/UserEntityChart.tsx:49]: Nit: use `terminology` (small t) for consistency with rest of code.

File: listenbrainz/webserver/static/js/src/stats/UserTopEntity.tsx (hunk 1 above)
Comment (jivteshsingh) [reply listenbrainz/webserver/static/js/src/stats/UserTopEntity.tsx:17]: Okk fine just updating in few mins

**Review by amCap1712 — APPROVED** [review]
LGTM. Thanks!

File: listenbrainz/webserver/static/js/src/stats/UserEntityChart.tsx [hunk 3]
```diff
@@ -264,6 +266,25 @@ export default class UserEntityChart extends React.Component<
   syncStateWithURL = async (): Promise<void> => {
     this.setState({ loading: true });
     const { page, range, entity } = this.getURLParams();
+
+    if (entity == "artist") {
```
Comment (amCap1712) [inline listenbrainz/webserver/static/js/src/stats/UserEntityChart.tsx:270]: Hi! Just saw CI, One more change needed. Replace all `==` here with `===`.

**Pull request merged**